### ISSUE TITLE
mailtrail doesnt trigger any campaigns after the last docker image update

### ISSUE THREAD

**Issue by ganeshkrishnan1** [Mailtrain-org/mailtrain#978]
I pulled the latest image last month from docker for the development version and none of the triggers have fired in the last one month.
The regular campaigns that are sent manually are working though.

Where do I even start to debug this issue?

**Comment by ganeshkrishnan1** [comment]
ok I can see this in the error logs
`{"log":"ERR! Senders Sending message to 3:1710 failed with error: Cannot read property 'replace' of null. Dropping the message.\n","stream":"stderr","time":"2020-11-06T01:53:55.458777676Z"}`

I assume it's trying to replace something in the sendMessage and is failing and then discarding the message

**Comment by ganeshkrishnan1** [comment]
I have found out that this problem happens when the merge field has null value and its used in the template.

**Comment by ericuldall** [comment]
```
169         if (value === undefined) { // in RSS it may happen that the key is present, but the value is undefined
170             return '';
171         }
```

Should this be `if (!value)` ? @bures

**Comment by ericuldall** [comment]
Nvm looks like this is fixed here: https://github.com/Mailtrain-org/mailtrain/blob/v2/server/lib/tools.js#L169

Suggest closing.

**Comment by bures** [comment]
@ericuldall - regarding the `value === undefined`, yes, it could also be `!value`. It would have a slighthly different semantics because for instance `0` or `''` are considered as false. Not that it would change much on the result. The problem is that we don't have a good set of requirements what the RSS may look like. So all this is just a kind of heuristics.

I'm fine with closing this. Please reopen it if the problem still persists.

**Comment by l4b4r4b4b4** [comment]
@bures 
For some reason, the automation does not seem to work with me. I Setup a Regular Campain and a few triggered ones with different triggers. Some with 0 days some with 1 day latency. As the regular campain stats show, I did open and click events get registered correctly. 
![grafik](https://user-images.githubusercontent.com/13406997/168566970-b7dbae57-6186-41dd-afed-b733cf6e8afe.png)

The logs show the following on open and click events:
```
mailtrain_1  | info HTTP GET /links/myiJWfKMhT/ZOZMJgfCuT/uRycuubMFo 200 1.241 ms - 42
mailtrain_1  | info HTTP GET /mosaico/img?src=https%3A%2F%2Flists.my-domain.de%2Ffiles%2Fcampaign%2Ffile%2F36%2Fa95a39816c7cf9069b2b5fe7dfe9b724&method=resize&params=534,null 200 3.405 ms - 40963
mailtrain_1  | info HTTP GET /links/myiJWfKMhT/ZOZMJgfCuT/uRycuubMFo/WjwIXEBfRk 302 2.495 ms - 94
```

So my first question would be: Is 0 day latency supported by mailtrain triggers?
And my second question: I will report back, if the 1 day triggers fired. But a test I did some weeks ago was not successfull on that either. So I am wondering if I am doing something wrong.

Cheers Luke
